Source/FlavoredNeutrinoContainerInit.cpp

```diff
@@ -36,6 +36,49 @@ Gpu::ManagedVector<GpuArray<Real,3> > uniform_sphere_xyz(int nphi_at_equator){
 	return xyz;
 }
 
+// residual for the root finder
+// Z needs to be bigger if residual is positive
```

Comment:
What does Z represent?

Reply:
Z is a parameter indicating how isotropic the distribution is. Z=0 means perfectly isotropic, and Z->infinity means neutrinos are all moving in one direction. So when Z increases, the flux factor increases. Ideally, one would write Z analytically in terms of f, but it is actually expressed as a transcendental equation. Since there is no way to do it analytically, we have to do the numerical root solve.  All of this is based on assuming that the angular distribution has this particular exponential form, which it in reality does not, but all of this just follows from assuming this form and following through with the math.

Reply:
I think it is better to cite the paper that explains Z than provide a detailed explanation in the comments. I did however make a change to better describe how to transcribe the equation in the Richers paper to the one here (i.e. "a" in the Richers paper is "Z" in the code). Do you think that is sufficiently descriptive?

Reply:
Yes, that makes things much clearer.